frontend/src/utils/DashboardUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { format, isToday, isTomorrow } from "date-fns";
+
+/* eslint-disable-next-line import/prefer-default-export */
+export const getAssistanceType = (isPickup: boolean) =>
```

Comment:
I'm also curious why this needed to be extracted into a function if it's online used once but I'm guessing we might use it again in the future?

Reply:
We get a nested ternary otherwise ^